server/autolinkplugin/plugin.go

```diff
@@ -120,14 +120,14 @@ func (p *Plugin) ProcessPost(c *plugin.Context, post *model.Post) (*model.Post,
 
 			channel, cErr := p.API.GetChannel(post.ChannelId)
 			if cErr != nil {
-				mlog.Error(cErr.Error())
+				p.API.LogError("Failed to get Channel", cErr.Error())
```

Comment:
`LogError` always expects an unequal number of arguments and will panic otherwise. Hence this has to be:
```suggestion
				p.API.LogError("Failed to get Channel", "error", cErr.Error())
```